File: WebScrapping-Backend/src/WebScrapping.Domain/Entities/FoodComposition.cs
namespace WebScrapping.Domain.Entities;

public class FoodComposition
{
    public int Id { get; set; }
    public string FoodCode { get; set; } = string.Empty;
    public string Component { get; set; } = string.Empty;
    public string? Unit { get; set; }
    public decimal? ValuePer100g { get; set; }
    public decimal? StandardDeviation { get; set; }
    public decimal? MinimumValue { get; set; }
    public decimal? MaximumValue { get; set; }
    public int? NumberOfDataUsed { get; set; }
    public string? Reference { get; set; }
    public string? DataType { get; set; }
    public Food? Food { get; set; }
}

File: WebScrapping-Frontend/src/types/foodComposition.ts
export interface Compositions {
  compositions: Composition[];
}

export interface Composition {
  id: number;
  foodCode: string;
  component: string;
  unit: string;
  valuePer100g: number;
  standardDeviation: number;
  minimumValue: number;
  maximumValue: number;
  numberOfDataUsed: number;
  reference: string;
  dataType: string;
}
